client/reader/lib/site-description-blacklist/test/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { isSiteDescriptionBlacklisted } from '../';
+
+describe( 'isSiteDescriptionBlacklisted', () => {
+	it( 'should return true if a site description is blacklisted', function() {
+		const blackListedDescription = 'Just another WordPress site';
+		expect( isSiteDescriptionBlacklisted( blackListedDescription ) ).to.equal( true );
```

Comment:
just curious, do you prefer `.to.equal( true )` over `.to.be.true` or `.to.be.ok`?